style.css

```diff
@@ -0,0 +1,9 @@
+body {
+  width: 80%;
+  font-family: 'IBM Plex Serif', serif;
+}
+
+/* font-family: 'VT323', monospace; The BEST FOOD BLOG
```

Comment:
zombie code 💀 Make sure to get rid of this before you make a pull request in the future.